.github/workflows/release-gh-pages.yml

```diff
@@ -24,8 +25,6 @@ jobs:
           npm ci
 
       - name: Build and push repositories json
-        env:
-          GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why removing `GITHUB_API_TOKEN`? It is needed in `build-repo-data.js`. I think you should populate it with your new `PAT_GITHUB`:

```
GITHUB_API_TOKEN: ${{ secrets.PAT_GITHUB }}
```

Reply:
or alternatively, keep using the default `GITHUB_TOKEN`, because it's a token needed for using GraphQL APIs so it doesn't need admin permissions:

```
GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
but isnt the context already set with the checkout action?

Reply:
`GITHUB_API_TOKEN` is a [custom env variable in the build script](https://github.com/ExpediaGroup/expediagroup.github.io/blob/main/src/scripts/build-repo-data.js#L39).

Reply:
got it. done